docs/configuration.md

```diff
@@ -141,6 +141,9 @@ params:
   # (optional) Define the subword marker. This is useful to apply noise at the
   # word level instead of the subword level (default: ￭).
   decoding_subword_token: ￭
+  # (optional) Whether subword_token is used as a spacer (as in SentencePiece) or a joiner (as in BPE).
+  # If ``None``, will infer  directly from :obj:`subword_token`.
+  is_spacer: False
```

Comment:
I propose to make the option name more specific, otherwise it could be confusing when appearing alone in the config. What about `decoding_subword_token_is_spacer`?